### PR TITLE
feat: Created Homepage of FakeSea 🚀

### PR DESCRIPTION
**Pull request by arjunbharti**
Created Homepage of **_FakeSea_**

- Header
- Main layout
- Trending products
- Buyer's guide
- Footer

### CODE REVIEW

**Review by sameep-s — APPROVED** [review]
Code looks good.

File: styles/homepage.css [hunk 1]
```diff
@@ -0,0 +1,361 @@
+header{
```
Comment (tushargoyal812) [inline styles/homepage.css:1]: use space before {  its good for indentation,use beautify extension for it in vs code

**Review by tushargoyal812 — COMMENTED** [review]
just take care of suggestion, looks good to merge

**Review by akash-1318 — COMMENTED** [review]
Looks good to merge

File: styles/homepage.css (hunk 1 above)
Comment (arjunbharti) [reply styles/homepage.css:1]: Sure, will take care of it!

**Pull request merged**